releasenotes/notes/fix-quantumerror-compose-7c54090a0cad0797.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed bug in the :meth:`qiskit.providers.aer.noise.QuantumError.compose`
```

Comment:
I don't think this should be in this PR, it's already in stable.

Reply:
The fix will also apply to the 0.5.0 release too. Normally reno is smart enough to figure out the duplicate (because it looks for the filename in the git log) but in this case it will show up twice because 0.4.1 doesn't use reno. That being said we we need a file in the releasnotes/notes dir so git will track it. We can just remove this at release time if you don't want it in the 0.5.0 release notes

Reply:
But this makes it sound like there is an extra bug in compose that wasn't fixed last time.

If you need to have a note there can you just add a release note that says we have switched to using reno for release notes?

Reply:
ok changed to a release note about release notes in: https://github.com/Qiskit/qiskit-aer/pull/642/commits/db12c394b7546643319b6d40fd8aedb2948d873b